### PR TITLE
Change the netfx compat pack to only ship stable packages, plus System.Drawing

### PR DESCRIPTION
**Pull request by mellinoe**
Also, I have removed the following packages because they are already in the reference set of .NET Core 2.0 and thus don't need to be in the compat pack:

* System.Numerics.Vectors
* System.Reflection.TypeExtensions
* System.ComponentModel.Annotations

Here is a gist of the resulting nuspec:

https://gist.github.com/mellinoe/31504172fa487acc94c62c5c6e5a0760

@weshaggard @danmosemsft

### CODE REVIEW

**Comment by mellinoe** [comment]
Test Linux x64 Release Build

**Comment by weshaggard** [comment]
Do you plan on adding the ServiceModel packages in another PR?

**Comment by danmosemsft** [comment]
@weshaggard for simplicity we were going to omit those from next week's release

**Comment by mellinoe** [comment]
@weshaggard We discussed it a bit more, and I went ahead and added the WCF dependencies into the project as well. We want them for the final release, and they are stable packages anyways, so no reason to leave them out now.

I added an "externalIndex.json" file right next to the pkgproj which is used to validate the stable WCF versions.

File: pkg/Microsoft.NETFramework.Compatibility/externalIndex.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "Packages": {
+    "System.ServiceModel.Primitives": {
+      "StableVersions": [
```
Comment (ericstj) [inline pkg/Microsoft.NETFramework.Compatibility/externalIndex.json]: Note: if you didn't want to apply baseline/pre-release to your dependencies you could have modified this: https://github.com/dotnet/corefx/blob/cf28f5105ac4bbcb9d19fa71a90fe4a219e0994b/pkg/dir.targets#L8

**Pull request merged**